### DISCUSSION TITLE
Welcome to dbterd Discussions!

### DISCUSSION THREAD

**Discussion by datnguye** [datnguye/dbterd#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pawan511** [comment]
I am trying to install this utility on dbt cloud 
By adding into package and running dbt deps
But I am getting an error

No dbt_project.yml found at expected path
![Uploading IMG_2622.jpeg…]()

**Reply by datnguye** [reply]
Hello @pawan511 , firstly thanks for raising the question! 
For your issue, `dbterd` is NOT the dbt package, but a companion Python package of the dbt project. Therefore, it should NOT be related to the dbt deps command or any of dbt commands. Please check the official documentation site here: https://dbterd.datnguyen.de/latest

Thanks

**Comment by pawan511** [comment]
Any suggestions to install it in dbt cloud

**Reply by datnguye** [reply]
Unfortunately there is not possible to do so.

Instead, you might want to follow up on this thread: https://github.com/dbt-labs/dbt-docs/issues/84#issuecomment-2571362703
